tcms/utils/tests/test_assign_permissions.py

```diff
@@ -0,0 +1,44 @@
+from django.contrib.auth.models import Group, Permission
+from django.test import TestCase
+
+from tcms.utils.permissions import assign_default_group_permissions
+
+
+class TestAssignDefaultGroupPermissions(TestCase):
+    """Test utils.permissions.assign_default_group_permissions()"""
+
+    def setUp(self):
+        self.admin = Group.objects.get(name='Administrator')
+        self.admin.permissions.all().delete()
+        self.tester = Group.objects.get(name='Tester')
+        self.tester.permissions.all().delete()
+
+    def test_administrator_has_all_permissions(self):
+        assign_default_group_permissions()
+        self.assertEqual(Permission.objects.all().count(),
+                         self.admin.permissions.count())
+
+    def test_tester_has_kiwi_apps_permissions(self):
+        assign_default_group_permissions()
+        for app_name in ['bugs', 'django_comments', 'linkreference', 'management',
+                         'testcases', 'testplans', 'testruns']:
+            self.assertEqual(set(Permission.objects.filter(
+                content_type__app_label__contains=app_name).values_list(
+                'codename', flat=True)),
+                set(self.tester.permissions.filter(
+                    content_type__app_label__contains=app_name).values_list(
```

Comment:
permissions.filter(pk__in=<query from line 25 without .values_list()>).exists() will return True (even if there is 1 object). 

combine with a count comparison and you guarantee that 1) the number of perms is the same 2) they are the same objects!